src/figuresClasses.ts

```diff
@@ -1,11 +1,78 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: 'red' | 'green' | 'blue';
 
-export class Triangle implements Figure {}
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape = 'triangle';
+
+  constructor(
+    public color: 'red' | 'green' | 'blue',
+    private a: number,
+    private b: number,
+    private c: number,
+  ) {
```

Comment:
[CODE KNOWLEDGE] - You don't need to manually assign constructor parameters to class properties when using access modifiers like `public` or `private`. TypeScript does this automatically. You can remove the assignment inside the constructor and just declare the parameters with their access modifiers.